integrations/stackit/pyproject.toml

```diff
@@ -0,0 +1,176 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "stackit-haystack"
+dynamic = ["version"]
+description = 'An integration of STACKIT as a StackitChatGenerator'
+readme = "README.md"
+requires-python = ">=3.8"
```

Comment:
Python 3.8 reached End Of Life (https://devguide.python.org/versions/), so I recommend supporting >=3.9.
Same for everything below.